randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -692,7 +722,16 @@ Extra - asset_id: 92
   * Layers: default
   * Normal Door to Noob Bridge/Door to Green Hill Zone
   > Door to Big Pink
-      Wave Beam
+      Any of the following:
+          Wave Beam
+          All of the following:
+              # Gate Glitch: https://youtu.be/_sZw9UQOG10
+              Gate Glitch (Beginner)
+              Missile ≥ 5 or Super Missile ≥ 5
+          All of the following:
+              # Gate Glitch (Advanced)
+              Gate Glitch (Advanced)
+              Missile or Super Missile
 
 > Pickup (Missile Expansion); Heals? False
```

Comment:
Going from this pickup to big pink is possible I assume with movement advanced and without morph?

Reply:
The tunnel crawl only works Left to Right for whatever reason. Its only possible in the other room because of the spikes.